src/internal/Subject.ts

```diff
@@ -14,17 +13,23 @@ import { arrRemove } from './util/arrRemove';
  * Subject, and you can call next to feed values as well as error and complete.
  */
 export class Subject<T> extends Observable<T> implements SubscriptionLike {
-  closed = false;
+  /** @internal */
+  _closed = false;
+
+  /**
+   * Will return true if this subject has been closed and is no longer accepting new values.
```

Comment:
I'd add docs somewhere to indicate that `Subject`s don't throw anymore if used after close and that that behavior is new as of V8.